src/AutoPilotPlugins/APM/APMSensorsComponent.qml

```diff
@@ -446,6 +446,34 @@ SetupPage {
                                 model:      _orientationsDialogShowCompass ? 3 : 0
                                 delegate:   singleCompassSettingsComponent
                             }
+
+                            QGCLabel {
+                                width:      parent.width
+                                visible:    globals.activeVehicle.sub && _orientationsDialogShowCompass
```

Comment:
The best way to handle visibility for these sort of things is to have the logic only in one place. Either with a single _showFoo property or with say an id on the label `magneticDeclinationLabel' and then the `Column` uses ``magneticDeclinationLabel.visible` to determine visibility. This prevent two separate sets of visibility logic from getting out of sync and causing bugs.

Reply:
ok, done!